README.md

```diff
@@ -110,25 +110,26 @@ Checkout our [Errors documentation][errors] to learn more.
+| `scope`         | String[] |**Optional** | List of permissions your application requires. This will default to requiring all scopes. The valid permission names are found in the [API Reference](https://smartcar.com/docs/api#get-all-vehicles). |
+| `test_mode`   | Boolean |**Optional** | Launch the Smartcar Connect in test mode. |
 
 ### `get_auth_url(self, force=False, state=None, vehicle_info=None)`
 
 Generate the Connect URL
 
 #### Arguments
-| Parameter       | Type | Description   |
-|:--------------- |:---|:------------- |
-| `force`   | Boolean |**Optional** Setting `forcePrompt` to `true` will show the permissions approval screen on every authentication attempt, even if the user has previously consented to the exact scope of permissions. |
-| `state`         | String |**Optional** OAuth state parameter passed to the redirectUri. This parameter may be used for identifying the user who initiated the request. |
-| `vehicle_info['make']`  | String |**Optional** Including the dict `vehicle_info` with a `make` property allows users to bypass the car brand selection screen. For a complete list of supported makes, please see our [API Reference](https://smartcar.com/docs/api#authorization) documentation. Makes are case-insensitive. |
-
+| Parameter       | Type | Required | Description   |
+|:--------------- |:---|:---|:------------- |
+| `force`   | Boolean |**Optional** | Setting `forcePrompt` to `true` will show the permissions approval screen on every authentication attempt, even if the user has previously consented to the exact scope of permissions. |
+| `state`         | String |**Optional** | OAuth state parameter passed to the redirectUri. This parameter may be used for identifying the user who initiated the request. |
+| `vehicle_info['make']`  | String |**Optional** | Including the dict `vehicle_info` with a `make` property allows users to bypass the car brand selection screen. For a complete list of supported makes, please see our [API Reference](https://smartcar.com/docs/api#authorization) documentation. Makes are case-insensitive. |
+| `single_select` | Boolean or Dictionary | **Optional** | An optional value that sets the behavior of the grant dialog displayed to the user. If set to `true`, `single_select` limits the user to selecting only one vehicle. Defaults to `false`. See the [Single Select guide](https://smartcar.com/docs/guides/single-select/) for more information.
+| `single_select.vin` | String | **Optional** | The VIN (Vehicle Identifier Number) of a vehicle. By passing in `single_select` as a dict with a `vin` property, the single select behavior will be turned on and Smartcar will authorize only the vehicle with the specified VIN. The `single_select_vin` takes precedence over the `make` parameter. Please refer to the [Single Select guide](https://smartcar.com/docs/guides/single-select/).
```

Comment:
This looks perfect, apart from the first sentence. Could we replace the first sentence with something like:
**An optional value that sets the behavior of the grant dialog displayed to the user. The `single-select` parameter takes a VIN (vehicle identification number) as a property.**